src/parsec_security/parsec_threat_model/threat_model.md

```diff
@@ -359,29 +359,29 @@ endpoint, mimicking the SPIFFE implementation's one.
+| ID | Justification                                                                                                                                                                                        | Details                                                                                                                                                                                                             |
+|----|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| 0  | Parsec uses a thread pool with a configurable amount of threads to limit the number of concurrent operations it can support so that the machine is not overloaded.                                   | [Done](https://github.com/parallaxsecond/parsec/commit/e87984c4d3eaafa1a1cb977dceadf773aca1f7db)                                                                                                                    |
+| 1  | Response statuses are audited to not give too much information to the client.                                                                                                                        | [Done](https://github.com/parallaxsecond/parsec-interface-rs/issues/59)                                                                                                                                             |
+| 2  | *(not DRCT)* The authenticators should only allow authentication tokens that are secure enough that brute-force attacks are not feasible.                                                            | *(JWTS)* The [algorithms](https://github.com/spiffe/spiffe/blob/master/standards/JWT-SVID.md#21-algorithm) prescribed by the JWT-SVID standard are secure enough to prevent brute-force attacks in most cases.      |
+| 3  | Listener implementations use an IPC mechanism respecting confidentiality and integrity of messages transmitted between the clients and the service (once the initial connection has been made).      | Unix Domain Socket Listener: the sockets used on the client and service side for the communication are represented by file descriptors that are only accessible by those processes.                                 |
+| 5  | Parsec is coded with safety in mind and is tested extensively.                                                                                                                                       | [Done](https://github.com/parallaxsecond/parsec-book/issues/35)                                                                                                                                                     |
```

Comment:
Hmm, I guess rebasing would confuse someone who already knew the threat model, but otherwise it would be ok to make `M-5` => `M-4`, `M-6` => `M-5` and so on

Reply:
Yes I did not do it because of other ressources that might point to specific numbers :/ Also does not seem that it matters too much.